client/src/Components/Pages/Search/index.js

```diff
@@ -88,8 +95,23 @@ export default class Search extends Component {
     });
   };
 
+  setActiveTab = e => {
+    const { tab } = e.target.dataset;
+    this.setState({ activeTab: tab });
+
+    if (tab === "recent") {
+      this.filterRecentReviews();
+    }
+  };
+
+  filterRecentReviews = () => {
+    const { category, sortedOrgs, recentReviews } = this.state;
+    const hasReviews = sortedOrgs[category].filter(org => org.totalReviews > 0)
+    this.setState({ recentReviews: { ...recentReviews, [category]: hasReviews }})
```

Comment:
```js
    const hasReviews = sortedOrgs[category].filter(org => org.totalReviews > 0)
    this.setState({ recentReviews: { ...recentReviews, [category]: hasReviews }})
```

could you please fix your eslint config, it seems that it deletes the semi-colons and it doesn't fix the indentation
here is when i save the file
```js
    const hasReviews = sortedOrgs[category].filter(org => org.totalReviews > 0);
    this.setState({
      recentReviews: { ...recentReviews, [category]: hasReviews },
    });
```
so I got a lot of changes that aren't related to my work

Reply:
Done :)